MapboxNavigation/RatingControl.swift

```diff
@@ -0,0 +1,142 @@
+import UIKit
+import CoreGraphics
+
+typealias RatingClosure = (Int) -> Void //rating
+
+/*@IBDesignable*/
+class RatingControl: UIStackView {
+    
+    //MARK: Constants
+    static let defaultSize = CGSize(width: 32.0, height: 32.0)
+    private let starTemplate = UIImage(named: "star", in: .mapboxNavigation, compatibleWith: nil)
+    
+    //MARK: Properties
+    private var stars = [UIButton]()
+    
+    var didChangeRating: RatingClosure?
+    
+    var rating: Int = 0 {
+        didSet {
+            updateSelectionStates()
+            didChangeRating?(rating)
+        }
+    }
+    
+    @objc dynamic public var selectedColor: UIColor = #colorLiteral(red: 0.1205472574, green: 0.2422055006, blue: 0.3489340544, alpha: 1) {
+        didSet {
+            updateSelectionStates()
+        }
+    }
+    @objc dynamic public var normalColor: UIColor = #colorLiteral(red: 0.8508961797, green: 0.8510394692, blue: 0.850877285, alpha: 1) {
+        didSet {
+            updateSelectionStates()
+        }
+    }
+    
+    @objc dynamic public var starSize: CGSize = defaultSize {
+        didSet {
+            configureStars()
+        }
+    }
+    
+    @objc dynamic public var starCount: Int = 5 {
+        didSet {
+            configureStars()
+        }
+    }
+    
+    //MARK: Initializers
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureStars()
+    }
+    
+    required public init(coder: NSCoder) {
+        super.init(coder: coder)
+        configureStars()
+    }
+    
+    //MARK: Private Functions
+    private func configureStars() {
+        removeStars()
+        addStars()
+        updateSelectionStates()
+    }
+    
+    private func addStars() {
+        for index in 0..<starCount {
+            let button = UIButton(type: .custom)
+            button.setImage(starTemplate, for: .normal)
+            button.adjustsImageWhenHighlighted = false
+            addButtonSizeConstraints(to: button)
+            
+            let setRatingNumber = NSNumber(value: index + 1)
+            let setRatingString = NumberFormatter.localizedString(from: setRatingNumber, number: .none)
+            let localizedStarLabel = NSLocalizedString("Set %@ star rating", comment: "Accessibility Star Label")
```

Comment:
Missing bundle parameter

Reply:
- [ ] do that thing that you forgot to do the last time